From 073bde7cc157e6c8da6aca029d4cc68afe9be9cf Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Fri, 17 Jan 2020 17:44:49 +0900
Subject: [PATCH 1/2] makefile: update dist-clean task

This patch is to add drivers to dist-clean task.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 GNUmakefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/GNUmakefile b/GNUmakefile
index 274f709..1e7ea9f 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -25,11 +25,11 @@ DOC_ROOT = docs/guides
 dist-clean:
 	make clean
 	rm -rf $(wildcard src/*/$(RTE_TARGET))
+	rm -rf $(wildcard src/drivers/*/$(RTE_TARGET))
 	rm -rf $(wildcard src/*/*.pyc)
 	rm -rf $(wildcard src/*/__pycache__)
 	rm -rf $(wildcard src/*/shared)
-	rm -rf $(wildcard src/mirror/vf)
-	rm -rf $(wildcard src/pcap/vf)
+	rm src/drivers/vhost/_postclean
 
 .PHONY: doc
 doc: doc-all
-- 
2.17.1

